static/app/views/dashboards/widgetBuilder/utils.tsx

```diff
@@ -144,15 +144,18 @@ export const generateOrderOptions = ({
   return options;
 };
 
-export function normalizeQueries({
-  displayType,
-  queries,
-  widgetType,
-}: {
-  displayType: DisplayType;
-  queries: Widget['queries'];
-  widgetType?: Widget['widgetType'];
-}): Widget['queries'] {
+export function normalizeQueries(
+  organization,
+  {
+    displayType,
+    queries,
+    widgetType,
+  }: {
+    displayType: DisplayType;
+    queries: Widget['queries'];
+    widgetType?: Widget['widgetType'];
+  }
+): Widget['queries'] {
```

Comment:
It might be better to just add the parameter to the object because that way we can guarantee accessing it by its key rather than relying on positional arguments. You might need to import the organization type if it's not already imported.

```suggestion
export function normalizeQueries(
  {
    displayType,
    queries,
    widgetType,
    organization
  }: {
    displayType: DisplayType;
    queries: Widget['queries'];
    widgetType?: Widget['widgetType'];
    organization?: Organization
  }
): Widget['queries'] {
```